doc/lua_api.txt

```diff
@@ -2195,6 +2195,15 @@ Two functions are provided to translate strings: `minetest.translate` and `minet
 
   this will be displayed as "Laine Rouge" on clients with a French locale.
 
+### Operations on translated strings
+
+The output of `minetest.translate` is a string, with escape sequences adding additional information to that string
+so that it can be translated on the different clients. In particular, you can't expect operations like string.length
+to work on them like you would expect them to, or string.gsub to work in the expected manner. However, string
+concatenation will still work as expected (note that you should only use this for things like formspecs; if you are
```

Comment:
`[...]; do not translate sentences by breaking them into parts, arguments should be used instead)`